Geodatenbezug/Processing.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Geodatenbezug.Models;
+using Microsoft.Extensions.Logging;
+
+namespace Geodatenbezug;
+public class Processing(IGeodiensteApi geodiensteApi, ILogger<Processing> logger)
+{
+    public async Task<List<Topic>> GetTopicsToUpdate()
+    {
+        var topics = await geodiensteApi.RequestTopicInfoAsync();
+        var currentTime = DateTime.Now;
+        var topicsToProcess = topics.FindAll((topic) =>
```

Comment:
PP: Wenn nur ein Parameter im Lambda steht, braucht es keine Klammern rundherum
```suggestion
        var topicsToProcess = topics.FindAll(topic =>
```